### PR TITLE
[RedisInsight] Document system requirements for RI.

### PR DESCRIPTION
**Pull request by Arushi201296**
**DO NOT MERGE without explicit approval**

- Documented recommended system requirements

Screenshot from 2021-02-01 12-41-18

- Linked to the above from within the installation page

Screenshot from 2021-02-01 12-40-53

### CODE REVIEW

File: content/ri/installing/system-requirements.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- No software pre-requisites
+
+{{< note >}}
+RedisInsight is supported on Mac hardware with Intel chips, but not for Mac hardware with the Apple M1 (ARM) chip
+{{< /note >}}
+
+## Linux
+
+- Ubuntu 18.04 LTS or later
+- 15GB Memory (RAM)
```
Comment (nishantgeorge) [inline content/ri/installing/system-requirements.md:31]: Why 15GB on Linux?

File: content/ri/installing/system-requirements.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Windows
+
+- Windows 10
+- 8GB Memory (RAM)
+- 32 bit or 64 bit (Application is 32 bit so it will work on both)
+- No software pre-requisites. RI should install and run on a fresh Windows system, no need to install any .NET framework or anything like that.
+
+## Mac
+
+- Max OS X 10.9 or later
```
Comment (nishantgeorge) [inline content/ri/installing/system-requirements.md:19]: I think this should be 10.13 or later. See [here](https://docs.redislabs.com/latest/ri/installing/install-redis-desktop/).

**Review by nishantgeorge — CHANGES_REQUESTED** [review]

File: content/ri/installing/system-requirements.md (hunk 2 above)
Comment (kaitlynmichaelredislabs) [reply content/ri/installing/system-requirements.md:19]: @nishantgeorge these are the requirments for the desktop client, correct? Would it make more sense for them to be included in the installation instructions?

Comment (Arushi201296) [reply content/ri/installing/system-requirements.md]: @kaitlynmichaelredislabs We have linked the recommended system requirements from within the installation instructions. 
The intention is to not overcrowd the installation instructions with other information.

**Review by nishantgeorge — APPROVED** [review]

**Review by kaitlynmichaelredislabs — CHANGES_REQUESTED** [review]
@Arushi201296 @nishantgeorge is there a reason the system requirements have to be on a new page? Could I add them to the installation procedure?

**Comment by K-Jo** [comment]
@kaitlynmichaelredislabs I have no opinion on this.  As long as the user finds it back appropriately.

**Comment by kaitlynmichaelredislabs** [comment]
I will incorporate these changes in my existing revision of the desktop client installation procedure (https://github.com/RedisLabs/redislabs-docs/pull/1211).

**Comment by nishantgeorge** [comment]
Closing in favour of https://github.com/RedisLabs/redislabs-docs/pull/1211

**Pull request closed without merging**